doc/source/cluster/kubernetes/k8s-ecosystem/volcano.md

```diff
@@ -311,7 +314,7 @@ kubectl get pods
 
 # NAME                                            READY   STATUS         RESTARTS   AGE
 # test-cluster-1-worker-worker-n5g8k              1/1     Running        0          9m4s
-# test-cluster-1-head-6668q                       1/1     Running        0          9m4s
+# test-cluster-1-head                             1/1     Running        0          9m4s
 # test-cluster-1-worker-worker-6tzf7              1/1     Running        0          9m4s
 ```
```

Comment:
There is no trailing hash on the head pod.
<img width="667" alt="volcano-trailing-hash-02" src="https://github.com/user-attachments/assets/0cfe3258-0c39-4a19-a1ba-72bb447cd924" />